app/routes/index.js

```diff
@@ -310,39 +265,86 @@ const storage = multer.diskStorage({
 			else
 				res.redirect('/admin');
 		});
-	app.post('/adduser', upload.single('profile_pic'), async function(req,res){
+	app.route('/adduser').post(adminAuth, async function(req,res){
 		try {
-			const user = new User(req.body);
-			console.log(user)
-			if(req.file){
-				user.profile_pic = req.file.tempFilePath;
-			}
-			await user.save()
-			res.json("User added successfully");
+			const form = new formidable.IncomingForm();
+
+    	form.parse(req, async function(err, fields, files){
+		console.log(fields)
+		console.log(files)
```

Comment:
I believe these were debugging aids? If so please strip these `console.log` statements before the final merge.